storage/include/storage/db_types.h

```diff
@@ -22,7 +22,7 @@ inline namespace v1DirectKeyValue {
 namespace detail {
 
 enum class EDBKeyType : std::uint8_t {
-  E_DB_KEY_TYPE_FIRST = 1,
+  E_DB_KEY_TYPE_FIRST = 65,
```

Comment:
Could you please explain the rationale for the value of `65`?

Reply:
it's in "visible" ascii range ;-)